tests/test_categorical_encoder.py

```diff
@@ -10,6 +10,7 @@
 from feature_engine.categorical_encoders import WoERatioCategoricalEncoder
```

Comment:
could we move the tests to a separate script called test_decision_tree_encoder.py?